### PR TITLE
Fix deploy step

### PR DESCRIPTION
**Pull request by cliveb**
"firebase init" options changed from 1 to 3. The doc only has 1. The last option for hosting changed from  'build/bundled' / build/unbundled'  to now only require build. Otherwise the deploy will fail, no matter how many times you try. Fix doc. See fix.

### CODE REVIEW

File: docs/tutorials/deploy.md [hunk 1]
```diff
@@ -25,10 +25,13 @@ The instructions below are based on the [Firebase quick start][Firebase quick st
     in the root of your project directory. Otherwise, go to
     [Firebase console][Firebase console] to create a new app.
     
-1.  When you initialize your app, you will be prompted for a directory to 
-    use as the public root (default is "public"). Enter `build/bundled` 
-    or `build/unbundled` (supports HTTP/2). `build` contains everything 
-    your application needs to run.
+1.  When you initialize your app, you will be prompted for
+- Database: space
```
Comment (ozasadnyy) [inline docs/tutorials/deploy.md]: You need db be enabled

Comment (cliveb) [reply docs/tutorials/deploy.md]: Ok, I'll double check Friday and adjust (v1.2 seems to work w/o db enabled).

**Comment by cliveb** [comment]
@ozasadnyy my struggles with release v1.2. Build by local nor build by Travis work. Looks like Firebase changed or but doc did not or something is wrong with custom-polymer build. Please can you assign someone to look what is wrong with the stable release who can solve and provide correct doc? 

https://github.com/gdg-x/hoverboard/tree/v1.2.0

Very urgent need help.

**Comment by ozasadnyy** [comment]
@cliveb show me your console output after running `gulp`

**Comment by cliveb** [comment]
@ozasadnyy console output

Clives-MBP:hoverboard seebe$ gulp run build
[02:31:38] Using gulpfile ~/GitHub/hoverboard/gulpfile.js
/usr/local/lib/node_modules/gulp/bin/gulp.js:129
    gulpInst.start.apply(gulpInst, toRun);
                  ^

TypeError: Cannot read property 'apply' of undefined
    at /usr/local/lib/node_modules/gulp/bin/gulp.js:129:19
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:667:11)
    at startup (bootstrap_node.js:201:16)
    at bootstrap_node.js:626:3
Clives-MBP:hoverboard seebe$

**Comment by ozasadnyy** [comment]
Looks like something with your gulp. Check if you are using Gulp v4 https://github.com/gdg-x/hoverboard/blob/v1.2.0/package.json#L18

**Comment by cliveb** [comment]
@ozasadnyy any chance you can G+ HOA into my machine to see what is wrong?

**Comment by cliveb** [comment]
??

Clives-MBP:hoverboard seebe$ gulp -v
[02:40:54] CLI version 3.9.1
[02:40:54] Local version 4.0.0-alpha.2
Clives-MBP:hoverboard seebe$

**Comment by ozasadnyy** [comment]
I can try in ~10 hours. Have you tried https://github.com/gulpjs/gulp-cli/issues/84#issuecomment-272958709 ?

**Comment by cliveb** [comment]
Thanks for you help. I'll try the gulp fix in 7 hours. I need some beauty sleep now.

**Comment by cliveb** [comment]
@ozasadnyy Please join our HOA https://hangouts.google.com/call/1lvpAiFPGuWFtebeC5XtAAkE

**Comment by cliveb** [comment]
@xCatG figured out my build. There is a couple of doco errors that trip up a newb. I will submit corrections.

**Comment by ozasadnyy** [comment]
Hey @cliveb, I didn't merge this PR as you said that you will add additional changes. Any news there? :)

**Comment by cliveb** [comment]
Let me circle around back to you plan by tomorrow. The doc needs a tweak
because Firebase deploy options changed.

Terse from phone

**Pull request merged**